src/PAModel/MergeTool/Deltas/ChangeProperty.cs

```diff
@@ -73,6 +73,9 @@ public override void Visit(BlockNode node, ControlPath context)
                             return;
                         }
                     }
+
+                    // Property wasn't present in base
```

Comment:
> // Property wasn't present in base [](http://example.com/codeflow?start=19&length=35)

when does this happen ?

Reply:
This is if it was a default-valued property in the base, since we remove those from the IR.